firmware/application/apps/ui_playlist.cpp

```diff
@@ -157,6 +157,7 @@ void PlaylistView::toggle() {
 }
 
 void PlaylistView::start() {
+    thread_null = false;
```

Comment:
Why do you need this? Can you just test if `replay_thread == nullptr` instead?

Reply:
> Why do you need this? Can you just test if `replay_thread == nullptr` instead?

ty! will test

Reply:
for some unknown reason `replay_thread == nullptr` doesn't work, crashed.  
If you don't mind, i think we can go original way, 8bit is OK anyway, I guess (only if you don't mind)

Reply:
You shouldn't need another flag to tell if a pointer is null. We should figure out why it's crashing.